dev-docs/bidders/pgamssp.md

```diff
@@ -8,6 +8,7 @@ tcfeu_supported: false
 coppa_supported: true
 schain_supported: true
 floors_supported: true
+gpp_supported: true
```

Comment:
This is deprecated, please use `gpp_sids`. See https://docs.prebid.org/dev-docs/bidder-adaptor.html#submitting-your-adapter

Example:

```
gpp_sids: tcfeu, tcfca, usnat, usstate_all, usp
```